version.txt

```diff
@@ -0,0 +1,5 @@
+# Versioning of the NuGet package should be done in the following format: 
+# {major}.{minor}.{buildId}{suffix} where suffix should be either 
+# "-alpha", "-beta" or "" (empty) for respectively non-master-branches, master-branches
+# and release-versions (which is triggered once the prerelease tag is removed)
```

Comment:
It actually is when a release is published: the azure function that triggers the build does not check whether the prerelease **flag** is set